articles/storage/storage-explorer/vs-azure-tools-storage-manage-with-storage-explorer.md

```diff
@@ -146,7 +146,7 @@ Storage Explorer lets you connect to individual resources, such as an Azure Data
 | Queues           | Yes                | No                   | Yes (URL)                      | No                 |
 | Tables           | Yes                | No                   | Yes (URL)                      | No                 |
 
-Storage Explorer can also connect to a [local storage emulator](#local-storage-emulator) using the emulator's configured ports.
+Storage Explorer can also connect to a [local storage emulator](#local-storage-emulator) using the emulator's configured ports. Take a note that you cannot mount a Blob container using Storage Explorer.
```

Comment:
@azarboon Can you clarify what you mean by "mounting" a blob container?

Also, if making a note, I would suggest using the appropriate format:

```markdown
Storage Explorer can also connect to a [local storage emulator](#local-storage-emulator) using the emulator's configured ports. 

> [!NOTE]
> You cannot mount a Blob container using Storage Explorer.
```